lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -117,6 +23,19 @@
     border-radius: 4px;
     color: #444;
   }
+
+  .wrapper.-compact {
```

Comment:
Renamed this to -compact (:thumbsup:) but unfortunately had to change the wrapper in every slide, in another PR I'd really like to pull,

```
      <div className={`slideshow__slide ${this.props.isActive ? '-active fade-in': ''}`}>
        <div className="container">
          <div className="wrapper -compact">
```

Out of every slide because it's just repeated for all of them & makes changing something like this quite annoying

Regardless ready for a quick 2nd look @DFurnes